recipes-support/snapd/snapd-git.inc

```diff
@@ -0,0 +1,14 @@
+PR = "r0"
+
+SRC_URI = "git://github.com/snapcore/snapd.git;protocol=https;branch=release/2.58;destsuffix=git/"
```

Comment:
```suggestion
SRC_URI = "git://github.com/snapcore/snapd.git;protocol=https;branch=release/2.60;destsuffix=git/"
```
or maybe better add something like:
```
BASEPV = "2.60"
PV = "${BASEPV}+git${SRCPV}"
SRC_URI = "git://github.com/snapcore/snapd.git;protocol=https;branch=release/${BASEPV};destsuffix=git/"
```

Reply:
which means that the _git recipe in kirkstone branch is broken too

Reply:
Updated https://github.com/snapcore/meta-snapd/pull/41 and here